lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -96,6 +96,77 @@ message UIRecoveryCodesPrintClickEvent {
   string user_name = 1;
 }
 
+// Contains common metadata for Discover related events.
+message DiscoverMetadata {
+  // Uniquely identifies Discover wizard "session". Will allow to correlate
+  // events within the same Discover wizard run. Can be UUIDv4.
+  string id = 1;
+
+  // anonymized
+  string username = 2;
+}
+
+// Represents a resource type.
+enum DiscoverResource {
```

Comment:
Does it make sense to define twice the same enum?
Can't we just import a third file?

Reply:
I'll try to understand if we can re-use them.
There's clearly a lot of repetition.